frontend/src/app/components/enterprise/enterprise-trial-waiting/ee-trial-waiting.component.ts

```diff
@@ -26,21 +26,28 @@
 // See docs/COPYRIGHT.rdoc for more details.
 // ++
 
-import {Component, ElementRef} from "@angular/core";
+import {Component, ElementRef, OnInit} from "@angular/core";
 import {I18nService} from "app/modules/common/i18n/i18n.service";
 import {EnterpriseTrialService} from "app/components/enterprise/enterprise-trial.service";
 import {HttpClient} from "@angular/common/http";
 import {NotificationsService} from "core-app/modules/common/notifications/notifications.service";
-
+import {distinctUntilChanged} from "rxjs/operators";
+import {TimezoneService} from "core-components/datetime/timezone.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/d0785ebf35512164c44ddf32daf84f33cfcb489d/frontend/src/app/components/enterprise/enterprise-trial-waiting/ee-trial-waiting.component.ts#L36"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>